Sources/Testing/Events/Clock.swift

```diff
@@ -40,9 +40,15 @@ extension Test {
 #if !SWT_NO_UTC_CLOCK
       /// The wall-clock time corresponding to this instant.
       fileprivate(set) var wall: TimeValue = {
-        var wall = timespec()
-        timespec_get(&wall, TIME_UTC)
-        return TimeValue(wall)
+        var time = timespec()
```

Comment:
Nitpick: the diff is smaller if you don't rename the variable.

Reply:
I suppose, but I do feel that it makes the code less clear. Renamed back to reduce change.